DQM/HLTEvF/plugins/TriggerBxMonitor.cc

```diff
@@ -117,18 +127,30 @@ void TriggerBxMonitor::fillDescriptions(edm::ConfigurationDescriptions & descrip
   desc.addUntracked<edm::InputTag>( "l1tResults", edm::InputTag("gtStage2Digis"));
   desc.addUntracked<edm::InputTag>( "hltResults", edm::InputTag("TriggerResults"));
   desc.addUntracked<std::string>(   "dqmPath",    "HLT/TriggerBx" );
+  desc.addUntracked<bool>( "doOrbitVsBX", false );
+  desc.addUntracked<int>( "minLS", 134 );
```

Comment:
please do not use as default values those that are valid for a single use case; rather, the default should be to monitor all collisions BX and a reasonable range of LS